File: Python/Human.py
# Kelas Manusia
class Human():
    # Atribut Manusia:
    __nik = ""
    __nama = ""
    __jenis_kelamin = ""

    # Konstruktor
    def __init__(self, nik, nama, jenis_kelamin):
        self.nik = nik
        self.nama = nama
        self.jenis_kelamin = jenis_kelamin

    # Setter
    def set_nik(self, nik):
        self.nik = nik

    def set_nama(self, nama):
        self.nama = nama

    def set_jenis_kelamin(self, jenis_kelamin):
        self.jenis_kelamin = jenis_kelamin

    # Getter
    def get_nik(self):
        return self.nik

    def get_nama(self):
        return self.nama

    def get_jenis_kelamin(self):
        return self.jenis_kelamin

File: PHP/Human.php
<?php  

class Human 
{
    // Private atribut:
    private $nik = "";
    private $nama = "";
    private $jenis_kelamin = "";

    public function __construct($nik = '', $nama = '', $jenis_kelamin)
    {
        $this->nik = $nik;
        $this->nama = $nama;
        $this->jenis_kelamin = $jenis_kelamin;
    }

    // Setter
    // NIK
    public function set_nik($nik = '')
    {
        $this->nik = $nik;
    }
    // Nama
    public function set_nama($nama = '')
    {
        $this->nama = $nama;
    }
    // Jenis Kelamin
    public function set_jenis_kelamin($jenis_kelamin = '')
    {
        $this->jenis_kelamin = $jenis_kelamin;
    }

    // Getter
    // NIK
    public function get_nik()
    {
        return $this->nik;
    }
    // Nama
    public function get_nama()
    {
        return $this->nama;
    }
    // Jenis Kelamin
    public function get_jenis_kelamin()
    {
        return $this->jenis_kelamin;
    }

    // Destructor
    function __destruct()
    {
        
    }
}

?>